tests/api/reports-orders.php

```diff
@@ -125,6 +125,7 @@ public function test_reports_schema() {
 		$this->assertArrayHasKey( 'customer_id', $properties );
 		$this->assertArrayHasKey( 'net_total', $properties );
 		$this->assertArrayHasKey( 'num_items_sold', $properties );
+		$this->assertArrayHasKey( 'total_formatted', $properties );
```

Comment:
It might also be worth adding a specific test for the `total_formatted` with an order with a different currency.

Reply:
Sure I'll try to add one! Thanks for the review @louwie17